src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Binskim.win-x64.RTR.dll.sarif

```diff
@@ -790,13 +790,10 @@
           "rules": [
             {
               "id": "BA2002",
+              "name": "DoNotIncorporateVulnerableDependencies",
               "fullDescription": {
                 "text": "Binaries should not take dependencies on code with known security vulnerabilities."
               },
-              "helpUri": "https://github.com/microsoft/binskim/blob/main/docs/BinSkimRules.md#rule-BA2002DoNotIncorporateVulnerableDependencies",
-              "help": {
-                "text": "Binaries should not take dependencies on code with known security vulnerabilities."
-              },
```

Comment:
can you verify what happened?
this should not happen.

Reply:
I ran the re-baseline, will see why there is this change. The order of the json node changed.

Reply:
found the reason, 
not related to our change,
was the result of changing the AnalyzeCommand to use MultithreadedAnalyzeCommandBase. I did verified if change it back this will disappear.
[https://github.com/microsoft/binskim/pull/505/commits/f87857a97e7a5ad24af0b0fbcd97fd9ef57b1dc8](https://github.com/microsoft/binskim/pull/505/commits/f87857a97e7a5ad24af0b0fbcd97fd9ef57b1dc8)

Reply:
we have fixed the sdk, got the latest from main and re-ran the baseline, now these order changes are gone.